test/reductions/test_grid_search_classification.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from fairlearn.metrics.demographic_parity import DemographicParity
+from fairlearn.reductions.grid_search_classification import GridSearchClassification
```

Comment:
Can I have a pointer on how to clean these up? I know that @romanlutz has a way :-)

Basically, I'd like these to be
```python
from fairlearn.metrics import DemographicParity
from fairlearn.reductions import GridSearchClassification
```

Reply:
Found @romanlutz 's information about this in Teams